hw17/jest-test/calc.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from "chai";
+ import { Calculator } from "../src/calculator";
+ const calc = new Calculator();
+
+ describe("Jest Calculator Test", () => {
+   test("Correctly finds the sum of two positive numbers", () => {
+     expect(calc.addition(1, 3)).to.equal(4);
+   });
+   test("Correctly finds the difference of two positive numbers", () => {
+     expect(calc.subtraction(33, 3)).to.equal(30);
+   });
+   test("Correctly finds the  multiplication of two positive numbers", () => {
+     expect(calc.multiplication(12, 12)).to.equal(144);
+   });
+   test("Correctly finds the quotient of two positive numbers", () => {
```

Comment:
Please correct:
`Correctly finds the result of division of two positive numbers`
The same is for `mocha` tests.